tortoise/verifying.go

```diff
@@ -94,27 +144,25 @@ func (v *verifying) sumGoodBallots(logger log.Log, ballots []tortoiseBallot) (we
 	sum := weightFromUint64(0)
 	n := 0
 	for _, ballot := range ballots {
-		if !v.isGood(logger, ballot) {
+		rst := v.isGood(logger, ballot)
+		if rst == bad {
 			continue
 		}
-		v.goodBallots[ballot.id] = struct{}{}
-		sum = sum.add(ballot.weight)
-		n++
+		v.goodBallots[ballot.id] = rst
+		if rst == good {
+			sum = sum.add(ballot.weight)
+			n++
+		}
 	}
 	return sum, n
 }
 
-func (v *verifying) isGood(logger log.Log, ballot tortoiseBallot) bool {
+func (v *verifying) isGood(logger log.Log, ballot tortoiseBallot) goodness {
```

Comment:
can you add comment here that compare "can be good" and "good"? thanks

Reply:
Can you clarify the request please